WooCommerce/WooCommerceTests/ViewRelated/Orders/OrderListViewModelTests.swift

```diff
@@ -259,57 +258,10 @@ final class OrderListViewModelTests: XCTestCase {
         }
     }
 
-    func test_storing_error_shows_error_banner_with_precedence_over_other_state() {
```

Comment:
Should we retain this test? I think it's still useful to test that the error banner takes precedence over other states.

Reply:
Since we don't have any more variable state, because simple payments is always enabled, this test would look exactly the same as the one above!

Reply:
Oh! I overlooked that other test, good call. 👍